lib/util/git/onboarding-branch-cache.ts

```diff
@@ -1,3 +1,4 @@
+// istanbul ignore file
```

Comment:
```suggestion
```

Reply:
Added this to prevent the codecov errors, will remove this when I add tests for the added file.